pkg/cmd/taskrun/testdata/TestTaskRunDescribe_status_with_steps.golden

```diff
@@ -0,0 +1,10 @@
+Status
```

Comment:
Why status reflecting twice not sure

Reply:
Actually, as the result shown in https://github.com/tektoncd/cli/pull/1291#issuecomment-772595504, the first status is shown associated with the decoration. 😂
I'm not sure whether such UI is proper since users may hope to get a more concise result with `--status --steps`, or could we have a better design?